.github/actions/javascript/proposalPoliceComment/proposalPoliceComment.ts

```diff
@@ -65,21 +63,86 @@ async function run() {
     const assistantID = getInput('PROPOSAL_POLICE_ASSISTANT_ID', {required: true});
     const openAI = new OpenAIUtils(apiKey);
 
+    /* eslint-disable rulesdir/no-default-id-values */
+    const issueNumber = payload.issue?.number ?? -1;
+    /* eslint-disable rulesdir/no-default-id-values */
+    const commentID = payload.comment?.id ?? -1;
+
+    // DUPLICATE PROPOSAL DETECTION
+    if (isCommentCreatedEvent(payload)) {
+        console.log('Starting DUPLICATE PROPOSAL DETECTION Check');
+        const newProposalCreatedAt = new Date(payload.comment.created_at).getTime();
+        const newProposalBody = payload.comment.body;
+        const newProposalAuthor = payload.comment.user.login;
+        // Fetch all comments in the issue
+        console.log('Get comments for issue #', issueNumber);
+        const commentsResponse = await GithubUtils.getAllCommentDetails(issueNumber);
+        console.log('commentsResponse', commentsResponse);
```

Comment:
This is going to be a potentially huge chunk of logs. Let's keep it inside a collapsible details tag with `core.startGroup` and `core.endGroup`